BoxContentSDKSampleApp/shareExtension/ShareViewController.m

```diff
@@ -0,0 +1,80 @@
+//
+//  ShareViewController.m
+//  shareExtension
+//
+//  Created by Thuy Nguyen on 5/8/17.
+//  Copyright © 2017 Box. All rights reserved.
+//
+
+#import "ShareViewController.h"
+#import <BoxContentSDK/BOXContentSDK.h>
+#import "BOXSampleAppSessionManager.h"
+
+@interface ShareViewController () <BOXURLSessionManagerDelegate>
+
+@property (nonatomic, strong, readwrite) BOXContentClient *client;
+
+@end
+
+@implementation ShareViewController
+
+- (void)setUp
+{
+#warning Set the client ID and client secret that can be retrieved by creating an application at http://developers.box.com
+
+    [BOXContentClient setClientID:@"your_client_id" clientSecret:@"your_client_secret"];
+
+    [BOXContentClient oneTimeSetUpInExtensionToSupportBackgroundTasksWithDelegate:self rootCacheDir:[BOXSampleAppSessionManager rootCacheDir] sharedContainerIdentifier:@"group.com.box.ios.dev" completion:^(NSError *error) {
```

Comment:
This is the same container identifier that the Box app uses. Should the sample app use a different group?